src/components/RestUsers/AddNewUserForm.js

```diff
@@ -61,6 +79,13 @@ const AddNewUserForm = props => {
           />
           <Styled.Error>{errors.email}</Styled.Error>
         </Form.Item>
+        <Form.Item label="User Roles">
+          <Select value={values.role} onChange={changeValues}>
```

Comment:
this Select needs a `name` of `role`, also it won't let me put a comment on a line that you didn't add, so this is the next best place. The `evt` fired on this Select change is actually a 'string' instead of a 'synthetic dom evt' like it should be. I don't know why this is happening, but here's a janky solution for now: replace our `changeValues` with this code.
```
// the evt of the Select input on our form needs to be investigated
const changeValues = evt => {
  if (typeof evt == 'string') {
    onChange('role', evt);
    return;
  }
  const { name, value } = evt.target;
  onChange(name, value);
}
```
it might be worthwhile to throw that comment about investigation directly above that part of the form, on line 81.5. This will get it working, but we need to figure out why this is happening

Reply:
In place now for further experimenting.